src/bokeh/sphinxext/bokeh_enum.py

```diff
@@ -25,18 +25,18 @@
 
 The directive above will generate the following output:
 
-    .. bokeh-enum:: baz
-        :module: bokeh.sphinxext.sample
+.. bokeh-enum:: baz
+    :module: bokeh.sphinxext.sample
 
-        Specify a baz style
+    Specify a baz style
 
 Although ``bokeh-enum`` may be used explicitly, it is more often convenient in
 conjunction with the :ref:`bokeh.sphinxext.bokeh_autodoc` extension. Together,
 the same output above will be generated directly from the following code:
 
 .. code-block:: python
 
-    #: Specify a baz style
+    # Specify a baz style
```

Comment:
This is is intentional, the colon tells Sphinx that this is a docstring for the attribute

Reply:
I didn't know the meaning of the `:`. I will revert this.